charts/k8s-monitoring/README.md

```diff
@@ -724,6 +724,7 @@ The Prometheus and Loki services may be hosted on the same cluster, or remotely
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | traces.enabled | bool | `false` | Receive and forward traces. |
+| traces.grafanaCloudMetrics.enabled | bool | `true` | Generate host info metrics from trace data, used in Application Observability in Grafana Cloud. |
```

Comment:
```suggestion
| traces.grafanaCloudMetrics.enabled | bool | `true` | Generate host info metrics from telemetry data, used in Application Observability in Grafana Cloud. |
```

rationale: the connector is likely to incorporate support for other telemetry signals (like OTEL HTTP) in the future

Reply:
Would it make more sense to move this outside of the "traces" section in the values file and inside the "receivers" section? That's where we've been putting things that apply to all telemetry data gathered via otlp.